app/src/main/java/com/itsaky/androidide/adapters/SearchListAdapter.kt

```
@@ -17,82 +17,170 @@ package com.itsaky.androidide.adapters
+		text.text = match.line
+		text.tag = match
+		CompletableFuture.runAsync {
+			try {
+				val scheme = SchemeAndroidIDE.newInstance(text.context)
+				val sb = JavaHighlighter().highlight(scheme, match.line, match.match)
+				ThreadUtils.runOnUiThread {
+					if (text.tag === match) {
+						text.text = sb
+					}
+				}
+			} catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)